docs/pages/setup/reference/metrics.mdx

```diff
@@ -99,9 +99,12 @@ Teleport Cloud does not expose monitoring endpoints for the Auth Service and Pro
 | `backend_batch_write_seconds` | histogram | cache | Latency for backend batch write operations. |
 | `backend_read_requests_total` | counter | cache | Number of read requests to the backend. |
 | `backend_read_seconds` | histogram | cache | Latency for read operations. |
-| `backend_write_requests_total` | counter | cache | Number of write requests to the backend. |
+| `backend_requests` | counter | cache | Number of write requests to the backend. |
 | `backend_write_seconds` | histogram | cache | Latency for backend write operations. |
 | `cluster_name_not_found_total` | counter | Teleport Auth | Number of times a cluster was not found. |
+| `dynamo_requests_total` | counter | dynamodb | Number of requests to the DynamoDB API. |
```

Comment:
I think we can capitalize the "dynamodb" category the same way we do the product itself.